guides/common/modules/proc_preparing-cloud-init-images-in-rhv.adoc

```diff
@@ -26,12 +26,6 @@ ifndef::orcharhino,satellite[]
 ----
 # {client-package-install-el8} cloud-init
 ----
-** On {EL} 7:
-+
-[options="nowrap" subs="+quotes,attributes"]
-----
-# {client-package-install-el7} cloud-init
-----
```

Comment:
Same here.

Reply:
This only affects non-Satellite; see line 10+16. I am only deleting this for upstream+orcharhino (line 16).